packages/twenty-front/src/modules/workflow/workflow-steps/workflow-actions/email-action/components/image-bubble-menu/ImageBubbleMenu.tsx

```diff
@@ -0,0 +1,94 @@
+import { BubbleMenuIconButton } from '@/workflow/workflow-steps/workflow-actions/email-action/components/text-bubble-menu/BubbleMenuIconButton';
+import { StyledBubbleMenuContainer } from '@/workflow/workflow-steps/workflow-actions/email-action/components/text-bubble-menu/TextBubbleMenu';
+import { type Editor } from '@tiptap/core';
+import { useEditorState } from '@tiptap/react';
+import { BubbleMenu } from '@tiptap/react/menus';
+import {
+  IconAlignCenter,
+  IconAlignLeft,
+  IconAlignRight,
+  IconTrash,
+} from 'twenty-ui/display';
+
+type ImageBubbleMenuProps = {
+  editor: Editor;
+};
+
+export const ImageBubbleMenu = ({ editor }: ImageBubbleMenuProps) => {
```

Comment:
Sorry I am not sure how I was supposed to test this? I tried to add it to the text menu but it was flickering, not sure if it's because of how I did or an inherent issue. How did you test it?